style.css

```diff
@@ -875,3 +880,515 @@ ul {
   font-size: 18px;
   color: #fff;
 }
+
+/* Desktop version */
+@media screen and (min-width: 768px) {
+  body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    overflow-x: hidden;
+    background-color: #e5e5e5;
+  }
+
+  .logo {
+    font-size: 25px;
+    font-weight: 700;
+    line-height: 20px;
+    background-color: transparent;
+    margin-left: 10%;
+    margin-top: 2%;
+  }
+
+  .app-header {
+    display: flex;
+  }
```

Comment:
- [ ] It seems like the changes you've made have created some problems on the header of your portfolio as the menu icon is visible on the desktop version 👇
    ![image](https://user-images.githubusercontent.com/104968236/236435212-dc5563e0-15ff-4ae2-b539-233bb02c5fcf.png)